resources/windows/wix/condition.wxs.erb

```diff
@@ -18,11 +18,10 @@
         <![CDATA[Installed OR (VersionNT >= 600)]]>
       </Condition>
 
-      <!--
       <Condition Message="32 bit <%= settings[:product_name] %> is no longer supported on 64 bit Windows Operating Systems.">
-          <![CDATA[Installed OR VersionNT64]]>
+          <![CDATA[Installed OR Not VersionNT64]]>
```

Comment:
I believe it's there so that the conditions do not trigger if the product is already installed i.e. a maintenance, uninstall or repair.

I believe this reads;
`![CDATA[Installed OR Not VersionNT64]]` 

not (Already installed or 32bit)